HealthcareSystem/Backend/Model/MyDbContext.cs

```diff
@@ -17,6 +18,9 @@ public class MyDbContext : DbContext
         public DbSet<Admin> Admins { get; set; }
         public DbSet<Feedback> Feedbacks { get; set; }
         public DbSet<PatientCard> PatientCards { get; set; }
+        public DbSet<Ingredient> Ingridients { get; set; }
+        public DbSet<Drug> Drugs { get; set; }
+        public DbSet<DrugType> DrugTypes { get; set; }
```

Comment:
The Ingredient and DrugType classes don't have keys as far as I can see, can Entity Framework generate tables for them?